src/components/common/calendar/Calendar.js

```diff
@@ -143,7 +143,12 @@ function CalendarApp() {
         position: 'relative',
       }}
     >
-      <Button onClick={() => showScheduleModal()}>Add Event</Button>
+      <Button
+        className="calendar-event-button"
```

Comment:
I see this is where the magic happens.  Out of curiosity, Is this, intended to be more of a unique class name (only for this button?) or would it be shared for any future buttons on the calendar as they may seem fit?  Maybe just being nitpicky here but I'm thinking this class name is generic enough it might be used again